cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -22,9 +22,11 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-4091 nmstate
 
-// CNV-4133 import, upload, clone
+=== Storage
 
-// CNV-4134 virtctl upload
+* It is now possible to import, upload, and clone virtual machine disks into namespaces that are subject to CPU and memory resource restrictions.
+
+* The virtctl tool can now monitor server-side upload post-processing asynchronously and more accurately reports the status of virtual machine disk uploads.
```

Comment:
Nits:
- I think these bullet points should be phrased in a user-centric way ("You can now...")
- I'd split this second sentence into two sentences. 
- Should virtctl be in backticks?

Reply:
I believe those two clauses are related so have left them as one sentence. 
Otherwise updated as suggested.